db/migrate/20230215131812_create_arts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateArts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :arts do |t|
+      t.string :title
+      t.text :description
+      t.integer :price
+      t.string :location
+      t.integer :size
+      t.references :user, null: false, foreign_key: true
+      t.references :category, null: false, foreign_key: true
```

Comment:
Like I said, category doesn't exist yet so there's no reason to add it to your code now.
We'll just make a migration file to add it to Art later

Reply:
deleted it